worlds/sm/variaRandomizer/rando/GraphBuilder.py

```diff
@@ -164,6 +164,7 @@ def escapeTrigger(self, emptyContainer, graph, maxDiff, escapeTrigger):
                     availLocsByArea[itemLoc.Location.GraphArea].append(itemLoc.Location.Name)
             self.log.debug("escapeTrigger. availLocsByArea="+str(availLocsByArea))
             sm.objectives.setAreaFuncs({area:lambda sm,ap:SMBool(len(container.getLocs(lambda loc: loc.Name in availLocsByArea[area]))==0) for area in availLocsByArea})
+            sm.objectives.setPercentFuncs({percent:lambda sm,ap:SMBool(len(container.getUsedLocs(lambda loc: True)) >= percent) for percent in [25, 50, 75, 100]})
```

Comment:
If I understand this right, it's not actually percent, but that many locations? As in, it's checking for being able to reach 25, 50, 75 and 100 locations respectively?

Reply:
You're right: thats wrong. Its should really be a percentage of obtainable location checks otherwise things like "Nothing" Items will be counted. I'll fix that.